src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py

```diff
@@ -2078,7 +2078,7 @@ class BGPConfigDaemon:
                              ('icmo_ttl', 'ttl {}', handle_ip_sla_common),
                              ('icmp_tos', 'tos {}', handle_ip_sla_common),
                            ]
-    srv6_locator_key_map = [(['opcode_prefix', 'opcode_act', 'opcode_vrf'], '{no:no-prefix}opcode {} {} vrf {}', hdl_srv6_locator)]
+    srv6_locator_key_map = [(['opcode_prefix', 'opcode_act', 'opcode_data'], '{no:no-prefix}opcode {} {} {}', hdl_srv6_locator)]
```

Comment:
@LARLSN Does this change support all the other Endpoing behaviors? Could you please add details to the description about the usage for DT/X. How about uSID endpoint behaviors?

Reply:
I've added details in description. And current load way in frrcfgd can not recognize uSID config field in json file, so a locator for uSID will still be load as an ordinary locator.